tests/dagpair.p.test.ts

```diff
@@ -13,9 +14,9 @@ const sleep = (ms: number) => new Promise(r => setTimeout(r, ms))
 let p1: Peer
 let p2: Peer
 
-describe('sync IPLD DAG between IPFS lite peers', () => {
-  beforeAll(async () => {
-    jest.setTimeout(20000)
+describe('sync IPLD DAG between IPFS lite peers', async function() {
+  before(async () => {
+    this.timeout(5000)
```

Comment:
@carsonfarmer can't get `before` one to run in the browser tests, times out over 2s